README.md

```diff
@@ -1,14 +1,14 @@
-How to work on the tool on your local machine:
+## Installation and Setup for Local Development
 
-1. Make sure you have [Node.js](https://nodejs.org/en/) installed (I used v6.11.1).
+1. Install [Node.js](https://nodejs.org/en/) (I used v6.11.1).
```

Comment:
Maybe "If you don't have [node.js](...) already, install it. ... To check if you have node.js, open a terminal window and type `node --version`. If a number of the format #.#.# gets printed, you have node. If not, you don't.

I dunno. Something like that.